src/components/Sticky/Sticky.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { STICKY } from '../ThemeIdentifiers';
+import { classNames } from '@shopify/react-utilities/styles';
+import * as baseTheme from './Sticky.scss';
+
+export type Position = 'top' | 'bottom';
```

Comment:
@HiirenP Can you also please add left & right position as well

Reply:
But it will work for particular div only. and position is for like top for stickyHeader and bottom for stickyFooter. and so as this will occupy 100% width and most cases due to Responsiveness of components and elements in emgage-ui, i guess this will not gonna work.